spaces/S000103/properties/P000089.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000103
+property: P000089
+value: true
+refs:
+- zb: "0867.46001"
+  name: Functional analysis. 2nd ed. (W. Rudin, 1991)
+---
+
+$X$ is a convex subset of a locally convex topological vector space $\mathbb R^I$. Therefore Schauder-Tychonof fixed point theorem (Theorem 5.28 in {{zb:0867.46001}}) applies.
```

Comment:
```suggestion
$X$ is a compact convex subset of a locally convex topological vector space $\mathbb R^I$. Therefore Schauder-Tychonoff fixed point theorem (Theorem 5.28 in {{zb:0867.46001}}) applies.
```
I think the Schauder-Tychonoff theorems requires a compact convex set.

Aside: I did not see the space $\mathbb R^I$ in pi-base.  Seems it would be a good one to add.  I can write an issue for it.